include/fastdds_statistics_backend/StatisticsBackend.hpp

```diff
@@ -354,17 +354,22 @@ class FASTDDS_STATISTICS_BACKEND_DllAPI StatisticsBackend
     /**
      * @brief Get a dump of the database.
      *
+     * @param clear If true, clear all the statistics data of all entities.
+     *
      * @return DatabaseDump object representing the backend database.
      */
-    static DatabaseDump dump_database();
+    static DatabaseDump dump_database(
+            bool clear);
```

Comment:
```suggestion
    static DatabaseDump dump_database(
            bool clear = false);
```
Clearing the data is a kind of 'extra' to the dumping operation, I believe it is expected that it does not clear anything. So make it that the users have to specify the argument only when they want to do the odd thing. This also makes them think twice before setting the clear to true.

Reply:
We discussed it and come to the conclusion of leave at it is.